templates/components/calendar-tab.html.twig

```diff
@@ -26,31 +26,28 @@
     'Feuillaison',
     'Floraison',
     'Fructification',
-    'Scénescence'
+    'Sénescence'
```

Comment:
You sould store these labels in class constant.
Twig usage be like :
`{{ set stades_labels = constant('App\\Entity\\Evenement::display_labels) }}`